app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -57,7 +93,9 @@ export const widgetDefinition = defineWidget([ArgumentPlaceholder, ArgumentAst],
   priority: 999,
   score: (props) => {
     const tags = props.input.info?.tagValues
-    if (tags == null) return Score.Mismatch
+    const [_, dynamicConfig] = props.config?.find(([name]) => name === props.input.info?.name) ?? []
```

Comment:
I'm not supper happy about that the `props.config` is an array that you have to scan through for each potentially matching widget. This might get expensive quickly. But I'm fine with it for now, we can improve on that in the future.